src/components/Footer.tsx

```diff
@@ -1,35 +1,138 @@
-import ChangeStory from "./ChangeStory";
-import { fineStyle } from "./styles";
+import { links } from "@/lib/constants/links";
+import { MentoLogoIcon } from "@/components/icons/mento-logo.icon";
+import { MobileAccordionMenu } from "@/components/mobile-accordion-menu";
+import { TwitterIcon } from "@/components/icons/twitter.icon";
+import { GithubIcon } from "@/components/icons/github.icon";
+import { DiscordIcon } from "@/components/icons/discord.icon";
 
-export default function Footer({ year }) {
+
+
+export const Footer = () => {
   return (
-    <footer className="box-border flex w-full max-w-[1280px] flex-wrap items-end justify-between px-[16px] pb-[24px]">
-      <div className="max-w-[380px]">
-        <div className="pb-[12px] text-[20px]">Mento Reserve</div>
-        <div className={fineStyle()}>
-          <strong>Disclaimer</strong> Nothing herein constitutes an offer to
-          sell, or the solicitation of an offer to buy, any securities or
-          tokens.
-        </div>
-        <div className={fineStyle("mt-[15px]")}>
-          © {year} AP Reserve Foundation
-        </div>
-      </div>
+    <>
+      <DesktopFooter />
+      <MobileFooter />
+    </>
+  );
+};
+
+const DesktopFooter = () => {
+  return (
+    <footer className="mx-auto mt-36 hidden items-start justify-between gap-16 border-t border-black px-4 pb-20 pt-20 dark:border-[#343437] lg:mx-10 lg:flex xl:mx-auto xl:max-w-[1120px] xl:gap-36">
       <div>
-        <a
-          className="ml-[10px] p-[10px]"
-          href="https://github.com/mento-protocol/reserve-site"
-        >
-          Source
-        </a>
-        <a className="ml-[10px] p-[10px]" href="/legal/terms">
-          Terms
-        </a>
-        <a className="ml-[10px] p-[10px]" href="/legal/privacy">
-          Privacy
-        </a>
-        <ChangeStory />
+        <MentoLogoIcon />
+        <p className="pt-3 font-inter text-[#636768] ">
+          Mento © 2024. <br />
```

Comment:
are we changing this from the prior dynamic `year` prop to be hardcoded to 2024 now? is that an intentional change?